perun-openapi/openapi.yml

```diff
@@ -6875,6 +6875,33 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/attributesManager/setAttributePolicyCollections:
+    post:
+      tags:
+        - AttributesManager
+      operationId: setAttributePolicyCollections
+      summary: "Sets attribute policy collections for an attribute definition provided in given policy collections."
```

Comment:
When I think about it, it might be reasonable to mention that original policy collections are deleted. Same for all the layers. What do you think?